src/main/java/com/edddoubled/orunmila/devpathroute/model/mapper/Mapper.java

```diff
@@ -0,0 +1,27 @@
+package com.edddoubled.orunmila.devpathroute.model.mapper;
+
+import com.edddoubled.orunmila.devpathroute.model.Department;
+import com.edddoubled.orunmila.devpathroute.model.dto.DepartmentDTO;
+import com.edddoubled.orunmila.devpathroute.model.dto.UserDTO;
+import com.edddoubled.orunmila.devpathroute.model.user.User;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.stereotype.Component;
+
+import java.util.Collections;
+import java.util.Optional;
+
+@Component
+// TODO
```

Comment:
you can use org.mapstruct, see [using-mapstruct-with-project-lombok](https://springframework.guru/using-mapstruct-with-project-lombok/)